bitnami/kubernetes-event-exporter/values-production.yaml

```diff
@@ -0,0 +1,183 @@
+## Global Docker image parameters
+## Please, note that this will override the image parameters, including dependencies, configured to use the global value
+## Current available global Docker image parameters: imageRegistry, imagePullSecrets
+##
+# global:
+#   imageRegistry: myRegistryName
+#   imagePullSecrets:
+#     - myRegistryKeySecretName
+
+## String to partially override kubernetes-event-exporter.fullname include (will maintain the release name)
+##
+# nameOverride:
+
+## String to fully override kubernetes-event-exporter.fullname template
+##
+# fullnameOverride:
+
```

Comment:
I think that we could include the `commonAnnotations` and `commonLabels` parameters to allow for custom annotation/labels across all deployed resources, what do you think?

Reply:
Sure, I implemented both.